.github/libs/GithubUtils.ts

```diff
@@ -404,19 +431,18 @@ class GithubUtils {
                 issue_number: issueNumber,
                 per_page: 100,
             },
-            (response) => _.map(response.data, (comment) => comment.body),
+            (response) => response.data.map((comment) => comment.body),
         );
     }
 
     /**
      * Create comment on pull request
      *
-     * @param {String} repo - The repo to search for a matching pull request or issue number
-     * @param {Number} number - The pull request or issue number
-     * @param {String} messageBody - The comment message
-     * @returns {Promise}
+     * @param repo - The repo to search for a matching pull request or issue number
+     * @param number - The pull request or issue number
+     * @param messageBody - The comment message
      */
-    static createComment(repo, number, messageBody) {
+    static createComment(repo: string, number: number, messageBody: string): Promise<RestEndpointMethodTypes['issues']['createComment']['response']> {
```

Comment:
Extract `RestEndpointMethodTypes['issues']['createComment']['response']` to a separate type

Reply:
Done!